gctscript/modules/gct/common.go

```diff
@@ -59,6 +59,26 @@ func WriteAsCSV(args ...objects.Object) (objects.Object, error) {
 		case indicators.OHLCV:
 			temp, err = convertOHLCV(args[i])
 			front = true
+		case "scriptContext":
```

Comment:
To be clear, when you're outputting to a CSV, you're saying that you can either provide the ctx _or_ a filename? It may be good to update the csv.gct commentary to highlight this